releases/release-1.21/release-notes/maps/pr-96691-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 96691
+releasenote:
+  text: Increase timeout for pod lifecycle test to reach pod status=ready
```

Comment:
I think this needs `do_not_publish: false` because it's not showing up in the markdown above